frontend/src/data/variables/CovidProvider.ts

```diff
@@ -36,6 +36,19 @@ class CovidProvider extends VariableProvider {
     this.acsProvider = acsProvider;
   }
 
+  // TODO- perhaps add checks to verify that MetricQuery is supported at all
```

Comment:
Could you do this in the superclass method using `allowsBreakdowns`? I think this would be nice to do in this PR because otherwise it will be very easy for it to go out of date as we add more datasets

Reply:
outdated